index.html

```diff
@@ -1,10 +1,2413 @@
+                        src="https://d.newsweek.com/en/full/1631397/cover-september-11-18-2020.jpg?w=170&q=95&f=87dabb57a32fecfaface1c7cf7a07d1f"
+                        alt="magazine thumb preview" />
+                      <img class="special-image"
+                        src="https://d.newsweek.com/en/full/1628669/cover-september-04-2020.jpg?w=170&q=95&f=1360030f30130d3440859fd318762628"
+                        alt="magazine thumb preview" />
+                    </div>
+                  </div>
+
+                  <div
+                    class="d-flex flex-column mx-auto align-items-center align-items-md-start align-items-lg-center justify-content-md-center ml-md-4 pl-md-0">
+                    <h5 class="subscribe-text font-weight-bold mt-4 text-wrap ">
+                      In-depth Stories You Don't See Elsewhere About Topics
+                      You Want to Read!
+                    </h5>
+                    <p class="subscribe-p text-secondary">All Access + Weekly Delivery.</p>
+                    <button type="button"
+                      class="btn-text bg-red border-0 btn-sm py-2 my-2 text-white text-center font-weight-bold text-uppercase">
+                      subscribe &nbsp; <i class="fas fa-chevron-right"></i>
+                    </button>
+                  </div>
+                </div>
+              </div>
+            </div>
+
+            <div class="col-lg-5">
+              <div class="newsletter position-relative mt-2 pt-3 pt-md-1 pt-lg-1 pb-lg-0 m-lg-0 border-bottom">
+
+                <h6 class="text-red font-weight-bold text-uppercase pb-1 mt-2 pt-3 pt-md-1 pt-md-2 p-lg-0">
+                  newsletter
+                </h6>
+                <div class="d-flex">
+                  <div class="d-none d-md-block col-md-3 col-lg-6 newsletter-image"></div>
+                  <div class="col-md-9 col-lg-6 align-items-center justify-content-md-center border">
+                    <div class="px-2 pt-4 pb-2 m-2 ">
+                      <h4 class="newsletter-heading p-md-0 font-weight-bold text-center text-md-left">The Starting 5
+                      </h4>
+                      <p class="text-secondary text-center text-md-left">
+                        See why nearly a quarter of a million subscribers begin
+                        their day with the Starting 5.
+                      </p>
+                      <input class="py-1 py-2 pl-2 border input-group" type="text" placeholder=" Email address" />
+                      <button type="button"
+                        class="btn-text-2 bg-red border-0 btn-sm btn-block py-2 my-2 text-white text-center font-weight-bold text-uppercase">
+                        free sign up &nbsp; <i class="fas fa-chevron-right"></i>
+                      </button>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] There is a break-line on the original page that is missing in your project. 
Please have a look. 
![Your Work](https://user-images.githubusercontent.com/58504655/93228844-64198700-f78f-11ea-8317-ca2e21dd3fb7.jpg)
![original work](https://user-images.githubusercontent.com/58504655/93228901-7693c080-f78f-11ea-8954-9a6b07aa9c78.jpg)